### PR TITLE
[gl-react-image] Add new package

### PR DESCRIPTION
**Pull request by martin-badin**
Discussion: #56186
Documentation: https://github.com/gre/gl-react-image

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test gl-react-image`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/gl-react-image/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Type definitions for gl-react-image 3.2
+// Project: https://github.com/gre/gl-react-image
+// Definitions by: Martin Badin <https://github.com/martin-badin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export interface GLImageProps {
+    center?: [number, number];
+    resizeMode: 'cover' | 'free' | 'contain' | 'stretch';
```
Comment (amcasey) [inline types/gl-react-image/index.d.ts:10]: From the docs and the source, it looks like `resizeMode` can be omitted and `'stretch'` will be used as a default.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/gl-react-image/index.d.ts (hunk 1 above)
Comment (martin-badin) [reply types/gl-react-image/index.d.ts:10]: you are right but 'contain' will be used as a default.

https://github.com/gre/gl-react-image/blob/master/src/index.js#L163

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by martin-badin** [comment]
Ready to merge

**Pull request merged**